tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php

```diff
@@ -0,0 +1,444 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Semicolon;
+
+use PhpCsFixer\Fixer\Semicolon\MultilineWhitespaceBeforeSemicolonsFixer;
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+use PhpCsFixer\WhitespacesFixerConfig;
+
+/**
+ * @author Egidijus Girčys <e.gircys@gmail.com>
```

Comment:
there seems to be code copied over from `NoMultilineWhitespaceBeforeSemicolonsFixerTest`, which is fine put please all copy over the authors of code;
```
 * @author John Kelly <wablam@gmail.com>
 * @author Graham Campbell <graham@alt-three.com>
 * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
```

Reply:
Sure thing!

Reply:
thanks! : D
and sorry for the yoda-comment, seems I can't write at the end of the work week ;)